### ISSUE TITLE
Alt-click on the floor - item handling problems

### ISSUE THREAD

**Issue by 00aquir** [yogstation13/Yogstation#10379]
- alt-clicking on the floor: no sprites, plus you cannot drag items at all (like paper bin into your hand (it just takes paper the moment you try dragging), etc.)
- for a full list of problems, look **more below**.

**Comment by redmoogle** [comment]
Only the sprites are broken, I can get drag working consistently

**Comment by 00aquir** [comment]
btw I also couldn't even pick up specifically the mech equipments from the floor into my hand by simple clicking (from alt-click menu), it worked only by right-click menu, or directly.

**Comment by 00aquir** [comment]
New round, same problem. Paper bin drag (from alt click menu, under "floor tab") picks up paper even when I'm starting to drag the whole thing with the mouse to my hand.

**Comment by redmoogle** [comment]
Alright that helps enough that I managed to replicate it.(the dragging issue, thought you meant ctrl-click interaction)

**Comment by 00aquir** [comment]
some more, plus the mentioned  problems:
- still now, there are **wrong or no sprites** loading for many items on the alt+click floor item list (some can load sometimes, and some don't, some loads wrong alt+click sprites, and sometimes only the right-click item list loads wrong sprites)
- because you **cannot drag items** with the mouse (onto yourself or your hand) from the alt+click floor item list, you cannot look inside a toolbox, or take one paper from a paper bin, because it just picks the whole container up
- the **item list resets** and jumps to the top if you try to look inside a container from the item list with alt+click
- **no animations** will show up (like a recharger charging a weapon) on the alt-click item list

basically, just compare the ui to the previous version.

**Comment by Sniblet** [comment]
If I use altclick once, move away and forget about it for a bit, then use it again, it pops open the tab and takes about a full second to update away from the contents of the tile I was checking before
Once in a while the altclick menu _freaks_ and shows a huge block of red and xeno blood on perfectly clean tiles, then followed by what's actually there, I think it's supposed to be an incredibly obnoxious error message of some kind

**Comment by 00aquir** [comment]
many of these problems still persist today, like often no sprites showing up, can't drag item from the alt+click floor item list (it just picks it up), etc.
